iec104.cpp

```diff
@@ -963,11 +1033,21 @@ void IEC104Client::m_addData(vector<Datapoint*>& datapoints, int64_t ioa,
         if (feature.value() == "ioa")
             measure_features->push_back(m_createDatapoint(feature.key(), ioa));
         else if (feature.value() == "value")
-            measure_features->push_back(
-                m_createDatapoint(feature.key(), value));
+        {
+            if (typeid(valuestate).name() == typeid(bool).name())
```

Comment:
this new code is not covered by branch, at least I don't see in code the test of the condition


https://sonarcloud.io/component_measures?metric=new_coverage&selected=fledge-power_fledge-south-iec104%3Aiec104.cpp&view=list&pullRequest=10&id=fledge-power_fledge-south-iec104